README.txt

```diff
@@ -706,6 +706,260 @@
+	options that are set are not the desirable ones.  
+
+	Options set for QC3.py, QC4_HV.py, QC4_SS.py, QC5_Gain.py, QC5_Rate.py scripts:
+
+	
+	-> QC3.py :   
+
+	--CanvTitleX=Time #left(s#right)
+	--CanvTitleY=Pressure #left(mbar#right)
+	--SelectColumnX=1
+	--SelectColumnY=2
+	--SelectRowStart=1
+	--SelectRowEnd=61
+	--CanvRangeX=0,3600
+	--CanvRangeY=0,35
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+
+	Moreover if the Fit option is used (example: python python/QC3.py —-file=QC3_GE11-X-L-CERN-0001_20170601.xlsm —-Fit)
+	the following options are set for the fit:
+
+	--Fit
+	--FitFormula=[0]*TMath::Exp([1]*x)
+	--FitParamIGuess=AMPLITUDE,6.2e-05
+	--FitRange=0,3600
+
+	-> QC4_HV.py :
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Applied Voltage #left(kV#right)
+	--SelectColumnX=3
+	--SelectColumnY=1
+	--SelectRowStart=3
+	--SelectRowEnd=37
+	--CanvRangeX=0,1000
+	--CanvRangeY=0,7
+	--YaxisScale
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+
+	Moreover if the Fit option is used (example: python python/QC4_HV.py —-file=QC4_GE11-X-L-CERN-0001_20170601.xlsm —-Fit)
+	the following options are set for the fit:
+
+	--Fit")
+	--FitFormula=[0]*x+[1]
+	--FitParamIGuess=Req,5   (where Req is the measured resistance from the excel file) 
+	--FitRange=0,1000
+
+	-> QC4_SS.py :
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Spurious Signal R_{SS} #left(Hz#right)
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+	--LatexLines=0.58,0.73, Readout~=~GEM3B
+	--SelectColumnX=3
+	--SelectColumnY=7
+	--SelectRowStart=3
+	--SelectRowEnd=37
+	--CanvRangeX=0,1000
+	--CanvRangeY=0,50
+	--SetErrY
+	--SelectColumnErrY=8
+
+	-> QC5_Gain.py :
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Effective Gain
+	--LatexLines=0.19,0.88, LS2~Detector~Production
+	--LatexLines=0.19,0.83, Gas~=~Ar/CO_{2}~#left(70/30#right)
+	--LatexLines=0.19,0.78, X-Ray~Target:~Ag
+	--LatexLines=0.19,0.73, X-Ray~V_{mon}~=~40~kV
+	--LatexLines=0.19,0.68, X-Ray~I_{mon}~~~=~5~#muA
+	--LatexLines=0.19,0.63, i#eta~=~4;~i#phi~=~2
+	--SelectColumnX=4
+	--SelectColumnY=11
+	--SelectRowStart=29
+	--SelectRowEnd=45
+	--CanvRangeX=0,750
+	--CanvLogXY=false,true
+	--CanvRangeY=10,1000000
+	--SetErrY
+	--SelectColumnErrY=12
+
+	-> QC5_Rate.py :
+
+	--CanvTitleX=Divider Current #left(#muA#right)
+	--CanvTitleY=Rate #left(Hz#right)
+	--LatexLines=0.19,0.88, LS2~Detector~Production
+	--LatexLines=0.19,0.83, Gas~=~Ar/CO_{2}~#left(70/30#right)
+	--LatexLines=0.19,0.78, X-Ray~Target:~Ag
+	--LatexLines=0.19,0.73, X-Ray~V_{mon}~=~40~kV
+	--LatexLines=0.19,0.68, X-Ray~I_{mon}~~~=~5~#muA
+	--LatexLines=0.19,0.63, i#eta~=~4;~i#phi~=~2
+	--SelectColumnX=4
+	--SelectColumnY=7
+	--SelectRowStart=29
+	--SelectRowEnd=45
+	--CanvRangeX=0,750
+	--CanvRangeY=0,1800
+	--SetErrY
+	--SelectColumnErrY=8
```

Comment:
Formatting needed, this is not really descriptive.  See the examples in [README.md](https://github.com/bdorney/CMS_GEM_Analysis_Framework/blob/develop/README.md).

For example if you give this to a student, from **only** this document, can they execute this and produce plots?